index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
💅 https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/lang

Reply:
What kind of change do I need to make here?

Reply:
it is a good comment